enterprise/models.py

```diff
@@ -2304,3 +2302,81 @@ def __repr__(self):
         Return uniquely identifying string representation.
         """
         return self.__str__()
+
+
+class PendingEnterpriseCustomerAdminUser(TimeStampedModel):
+    # pylint: disable=line-too-long
```

Comment:
nit: Do we need this? Can we do a line break for the long line?

Reply:
I wasn't sure if the PII annotations with a line break would causes problems on the PII annotation linting that happens, so I copied this pattern from an existing docstring with similar PII annotations. I'll give it a shot with a line break and see if CI fails, though.